Bukkit/src/main/java/com/plotsquared/bukkit/player/BukkitPlayerManager.java

```diff
@@ -66,13 +66,12 @@ public BukkitPlayer getPlayer(final @NonNull Player object) {
     }
 
     @Override
-    @SuppressWarnings("deprecation")
```

Comment:
I can't link the method above, but the suppression should be redundant there too.